applications/content-drafting.md

```diff
@@ -0,0 +1,29 @@
+Machine translation allows to quickly draft content in natural language. Applications:
+
+# Mock-up localization
+Raw machine-translated text is a good mock-up to verify how the target language appears in localized content such as:
+* user interface,
+* images and diagrams.
+
+MT produces natural-language content that shall not be verified for accuracy at this step, but presents such features as target alphabet and typical word/sentence length. 
+When applied to localized content, MT allows to quickly issues, for example:
```

Comment:
I think a verb is missing.

`When applied to localised content, machine translation allows to spot issues.`

Reply:
Spot was meant to be a verb, I now use "detect" instead to make it clear.

Reply:
"Spot" was supposed to be the verb, I now use "detect" for clarity.